### PR TITLE
Feature/update colour palette

### PR DESCRIPTION
**Pull request by NoelleDL**
### What

Update colour values based on new colour palette. 

### How to review

Go to chart config in Babbage and see old palette. Pull this branch and see new colour palette options (Primary, Extended, Alternate). 

### Who can review

@armstrongb and anyone else but me.

### CODE REVIEW

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 1]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
-		'rgba(0, 60, 87, 1)',			// dark blue/navy
-		'rgba(168, 189, 58, 1)',		// lime green
-		'rgba(39, 160, 204, 1)',		// light blue
-		'rgba(135, 26, 91, 1)',			// plum purple
-		'rgba(246, 96, 104, 1)',		// pink
-		'rgba(116, 108, 177, 1)', 		// light purple
-		'rgba(34, 208, 182, 1)'			// turquoise
+	var primary = [
+
+        #206095,    // blue
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:21]: Some of the colours listed have new ONS wide names, we should refer to them as these from now on.


```suggestion
        #206095,    // ocean-blue
```

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 2]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #27A0CC,    // light blue
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:22]: Sky-blue

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 3]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #003C57,    // dark blue/navy
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:23]: night-blue

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 4]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #118C7B,    // green
+        #A8BD3A     // lime green
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:25]: Spring-green

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 5]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
 	];
 
+	var extended = [
+
+        #206095,    // blue
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:30]: ocean-blue

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 6]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #27A0CC,    // light blue
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:31]: sky-blue

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 7]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #003C57,    // dark blue/navy
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:32]: night-blue

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 8]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #118C7B,    // green
+        #A8BD3A,    // lime green
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:34]: spring-green

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 9]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #871A5B,    // plum purple
+        #F66068,    // pink
+        #746CB1,    // light purple
+        #22D0B6     // turquoise
+    ];
+
+    var alternate = [
+
+        #206095,    // blue
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:43]: ocean-blue

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 10]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #27A0CC,    // light blue
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:44]: sky-blue

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars [hunk 11]
```diff
@@ -12,22 +12,46 @@ TODO: split out colours palette selection, axis alignment and positioning
...
+        #871A5B,    // plum purple
+        #A8BD3A,    // lime green
```
Comment (dandwal) [inline src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars:46]: spring-green

**Comment by armstrongb** [comment]
Not all chart colours have a 'new palette' name so some legacy names may remain.

**Review by armstrongb — CHANGES_REQUESTED** [review]
LGTM with Dan's comments to be addressed.

File: src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars (hunk 11 above)
Comment (NoelleDL) [reply src/main/web/templates/handlebars/highcharts/config/base/base-chart-config.handlebars]: Made these changes.

**Review by armstrongb — DISMISSED** [review]
LGTM

**Review by mwnicks — APPROVED** [review]
Reapproving following rebase

**Pull request merged**